src/index.js

```diff
@@ -0,0 +1,60 @@
+// /src/index.js - entry point
+
+import './styles.css';
+import UI from './modules/ui.js';
+import { validateDescription } from './modules/helpfulFunctions.js';
+
+// Event: on content load
+document.addEventListener('DOMContentLoaded', UI.displayTasks());
+
+// Event: when form is submitted
+document.querySelector('#newTask').addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  // Get form values
+  const taskDescription = document.querySelector('#taskDesc').value.trim();
+  if (validateDescription(taskDescription)) {
+    // Add task
+    UI.addTask(taskDescription);
+  }
+  document.querySelector('#taskDesc').value = '';
+});
+
+// Event: when icon is clicked to add task
+document.querySelector('#clickEnterIcon').addEventListener('click', () => {
+  // Get form values
+  const taskDescription = document.querySelector('#taskDesc').value.trim();
+  if (validateDescription(taskDescription)) {
+    // Add task
+    UI.addTask(taskDescription);
+  }
+  document.querySelector('#taskDesc').value = '';
+});
+
+// Event: click anything on taskList
+document.querySelector('#taskList').addEventListener('click', (e) => {
+  const classesIcn = e.target.parentElement.className;
+  const classesArr = classesIcn.split(' ');
+
+  const li = e.target.parentElement.parentElement.parentElement;
+  // console.log(li);
+  const index = li.dataset.id - 1;
+
+  // Event: when the three dots icon gets clicked
+  if (classesArr.indexOf('edtIcn') !== -1) {
+    UI.changeLiToEditMode(li);
+  }
+
+  // Event: when the check icon gets clicked to REMOVE
+  if (classesArr.indexOf('removeIcn') !== -1) {
+    UI.removeTask(index, li);
+  }
+
+  // Event: when the the trash icon gets clicked to UPDATE
+  if (classesArr.indexOf('acceptIcn') !== -1) {
+    const newDesc = document.querySelector('#inputEdit').value.trim();
+    if (validateDescription(newDesc)) {
+      UI.updateTask(index, newDesc, li);
+    }
+  }
```

Comment:
- [ ] Please implement the function to delete all checked items, as for now, when the user selects some tasks and tries to delete them all, it does not work. kindly fix this and you will be good to go

Reply:
@BenMukebo  are you on the correct branch it is working fine when i test it

Reply:
@BenMukebo that requirement is for day 4,,, this is day 3 submission. I am not sure i  am clear.

Reply:
@Kingstalux should i resubmit as it is ?

Reply:
Yeah @oshanedesign,
Resubmit for a review and it will be approved 👍

Reply:
@Kingstalux resubmitted. thank you

Reply:
> Hello @BenMukebo, I think for this project only the add, edit and delete functions should be implemented. The clear all completed method is for the next project

You are right, thanks @Kingstalux, I misunderstood this requirement.